contracts/upgradeable/LockableUpgradeable.sol

```diff
@@ -0,0 +1,80 @@
+// SPDX-License-Identifier: AGPL-3.0-only
```

Comment:
As discussed IRL, maybe we should just use the OZ lockable contract? Do you see any problem with that?

Reply:
Nope, I looked into Lockable. The main feature we add is the `_startReentrantGuardDisabled` internal function but that's not used by Across contracts so I think we're safe to use OZ lockable

Reply:
Awesome!